site/content/en/docs/api_sdk/server-api/_index.md

```diff
@@ -1,13 +1,37 @@
 ---
-title: 'REST API design principles'
-linkTitle: 'REST API design principles'
-weight: 100
-description: 'Information on using the REST API scheme and principles of its design.'
+title: 'Server API'
+linkTitle: 'API'
+weight: 2
+description: ''
 ---
 
-## REST API scheme
+## Overview
 
-Common scheme for our REST API is `<VERB> [namespace] <objects> <id> <action>`.
+CVAT server provides HTTP REST API for interaction. Each client application - be it
+a command line tool, browser or a script - all interact with CVAT via HTTP requests and
+responses:
+
+![CVAT server interaction image](/images/server_api_interaction.svg)
```

Comment:
I will prefer to have a console icon instead of a user and a list (python script?). Probably you want to ask Alexander to draw a picture for you after he comes back from vacation.